src/main/kotlin/no/nav/syfo/aktivitetskrav/cronjob/PublishExpiredVarslerCronJob.kt

```diff
@@ -0,0 +1,37 @@
+package no.nav.syfo.aktivitetskrav.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.aktivitetskrav.AktivitetskravVarselService
+import no.nav.syfo.application.cronjob.Cronjob
+import no.nav.syfo.application.cronjob.CronjobResult
+import org.slf4j.LoggerFactory
+
+class PublishExpiredVarslerCronJob(
+    private val aktivitetskravVarselService: AktivitetskravVarselService
+) : Cronjob {
+    override val initialDelayMinutes: Long = 2
+    override val intervalDelayMinutes: Long = 30
```

Comment:
Fikk jo litt svar fra deg @eirikdahlen hva du tenkte. Én gang om dagen høres litt sjeldent ut synes jeg🤔 Holder det med en gang hvert 30. minutt? 😅

Reply:
Tja, eller sånn jeg har forstått det så vil vel bare én av kjøringene per dag faktisk publisere, ettersom man ser på hele dager og ikke på tidsstempel? Da er det jo en hard cut, og den første cronjobben som kjører etter midnatt vil publisere alle som trengs å publiseres, og så vil resten bare ha `updated: 0` resten av dagen? 🤷🏼‍♂️ 

Det jeg tenkte mest på var om vi skulle ha egne tider for dev, sånn at det går litt raskere å teste dette. Feks kunne man hatt hver 4. time i prod og hvert 5. minutt i dev. Det er noe sånt vi har gjort i `isbehandlerdialog` for eksempel. 
Da legges det feks i env-variabler og tas inn i cronjobben, som her: https://github.com/navikt/isbehandlerdialog/pull/63

Reply:
True! Vi sjekker jo bare inneværende dag. Kanskje det da holder med 1 om dagen da? 

Jeg skal legge til egne tider for dev 😁